data/en/2020/default/registration.yml

```diff
@@ -1,3 +1,3 @@
-title: "Save the date!"
+title: "About the Event"
 end_date: 2020-09-16T23:59:00-00:00
```

Comment:
Please change the end date to December 9, 2020. The following day of the event.

Reply:
Have not made the changes for other languages, if you want I can also do for other lang.